gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Wed Oct 10 13:05:21 BST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-bin.zip
```

Comment:
@kittinunf [Q]: why haven't we upgraded to 4.10? On my latest PR, everything works with 4.10 🌊

Reply:
it will, unfortunately, fail on publishing task! >_<.

I think the fact that they don't fix this issue is a bit obnoxious as a user. I will migrate our publishing script to official [bintray](https://github.com/bintray/gradle-bintray-plugin)